lib/src/suggestor_mixins.dart

```diff
@@ -87,28 +77,25 @@ mixin AstVisitingSuggestor<R> on AstVisitor<R> implements Suggestor {
 /// Note that this mixin provides an implementation of [generatePatches] that
 /// should not need to be overridden except for performance optimization reasons
 /// like avoiding analysis on certain files.
-mixin ElementVisitingSuggestor<R> on ElementVisitor<R> implements Suggestor {
-  final _patches = <Patch>{};
+mixin ElementVisitingSuggestor<R> on ElementVisitor<R> {
+  final patches = <Patch>[];
 
   /// The context helper for the file for which patches are currently being
   /// generated.
   FileContext get context => _context;
   FileContext _context;
```

Comment:
These can be removed now, right?

Reply:
Yeah, I'm considering leaving it.. Now that I've worked around the potential issue you pointed out with the patch stream being lazyily generated, I might try to keep this in place and set it right before calling `accept()` (which will happen synchronously) so that the `FileContext` can be used in the visit methods. I noticed over_react_codemod uses that in a couple of places to get the URL of the file to determine if it's public.